packages/core/src/components/ModalNew/layouts/ModalBasicLayout/__stories__/ModalBasicLayout.stories.tsx

```diff
@@ -31,14 +30,17 @@ export default {
   subcomponents: { ModalBasicLayout, ModalHeader, ModalContent, ModalFooter, ModalFooterWizard, TransitionView },
   argTypes: metaSettings.argTypes,
   decorators: metaSettings.decorators,
-  tags: ["internal"]
+  tags: ["internal"],
+  parameters: {
+    chromatic: {}
+  }
 } satisfies Meta<typeof Modal>;
 
 export const Overview: Story = {
   decorators: [(Story, context) => withOpenedModalPreview(Story, { isDocsView: context.viewMode === "docs" })],
-  render: (args, { show, setShow }) => {
+  render: (args, { show, setShow, container }) => {
     return (
-      <Modal id="modal-basic" show={show} size="medium" onClose={() => setShow(false)} {...args}>
+      <Modal id="modal-basic" show={show} size="medium" onClose={() => setShow(false)} container={container} {...args}>
```

Comment:
Why do you need to pass it? Isn't it using the default document.body?

Reply:
I don't want it to default to document.body, I want it to be under the preview container, I have no choice but to expose it in the stories

Reply:
Got it
Would a decorator work for this? Because it's irrelevant for the stories